packages/nextjs-contentful-renderer/src/utils/contentful/getPageById/getPageById.ts

```diff
@@ -1,14 +1,15 @@
-import { ApolloClient, gql, NormalizedCacheObject } from '@apollo/client';
-import { Page } from './types';
+import { gql } from '@apollo/client';
+import { GetPageById, Page } from './types';
 
-export const getPageById = async (
-  apolloClient: ApolloClient<NormalizedCacheObject>,
-  pageId: string,
-  preview: boolean
-): Promise<Page | undefined> => {
+export const getPageById = async ({
+  apolloClient,
+  pageId,
+  preview,
+  skipTheme,
```

Comment:
why this?, why the client need to know if it has to request the theme for a page or not?, this does not makes any sense to me